src/docs/rts-new.md

```diff
@@ -0,0 +1,170 @@
+# Interaction with timestamps
+
+The main thing RTS is concerned with are the durable and stable
+timestamps. The high-level details are covered above, what follows is a more
+detailed view of how we use these timestamps.
+
+We read `txn_global` for pinned and stable timestamps at the start of RTS,
+and in diagnostic builds we read them again at the end to assert they didn't
+change underneath us (indicating the system was not quiescent, and therefore
+a bug).
+
+Once RTS has finished, we roll back the global durable timestamp to the
+stable timestamp, since if there are no unstable updates, `txn_global`'s
+durable timestamp can by definition be moved backwards to the stable
+timestamp. If we weren't to do this, it wouldn't automatically be a problem,
+but we'd likely end up unnecessarily scanning tables the next time RTS is
+run.
```

Comment:
The last sentence does not add much to me. "it wouldn't automatically be a problem" sounds like it may sometimes be a problem. I suggest just removing it and stating what we do.

Reply:
Thanks - I've reworded this section so that the motivation behind rolling back the global durable timestamp is clearer in 4f053fe.